lib/ansible/modules/storage/ibm/ibm_sa_host_ports.py

```diff
@@ -109,9 +109,14 @@ def main():
     port_exists = False
     ports = [port.get('port_name') for port in ports]
 
+    fc_ports = (module.params.get('fcaddress')
+                if module.params.get('fcaddress') else [])
+    iscsi_ports = (module.params.get('iscsi_name')
```

Comment:
```suggestion
    iscsi_ports = module.params.get('iscsi_name', [])
```

Reply:
@Akasurde , Same as above. 
modules.params.iscsi_name will be None here If it's not supplied in the playbook.
(different from [])